recipe/meta.yaml

```
@@ -12,28 +14,25 @@ source:
     - setup.py.patch
 
 build:
-  number: 1
+  number: 2
   skip: true  # [not linux]
-  skip: true  # [cuda_compiler_version == "None"]
-  skip: true  # [cuda_compiler_version == "10.2"]
-  skip: true  # [cuda_compiler_version == "11.0"]
+  skip: true  # [cuda_compiler_version != "11.6"]
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   build:
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
-    - setuptools
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
   host:
     - python
     - pip
-    - setuptools
+    - cudatoolkit {{ cuda_version }}.*
   run:
     - python
-    - pip
     - numba >=0.54
-    - numpy
+    - cudatoolkit >={{ cuda_major ~ ".0" }},<={{ cuda_version }}
+  run_constrained:
+    - __cuda >={{ cuda_major ~ ".0" }},<={{ cuda_version }}
```

Can change these to also rely on `cuda_compiler_version`

```suggestion
    - cudatoolkit >={{ cuda_major ~ ".0" }},<={{ cuda_compiler_version }}
  run_constrained:
    - __cuda >={{ cuda_major ~ ".0" }},<={{ cuda_compiler_version }}
```